hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -133,6 +133,10 @@
   private static final String MERGE_DATA_VALIDATION_CHECK_ENABLED = "hoodie.merge.data.validation.enabled";
   private static final String DEFAULT_MERGE_DATA_VALIDATION_CHECK_ENABLED = "false";
 
+  // Concats inserts to data files without merging
+  private static final String MERGE_ALLOW_DUPLICATE_INSERTS = "hoodie.merge.allow.duplicate.inserts";
+  private static final String DEFAULT_MERGE_ALLOW_DUPLICATE_INSERTS = "false";
```

Comment:
IMO, this could be defined as boolean type.

Reply:
just following the convention we have been using in hoodie so far.